jest.config.js

```diff
@@ -1,20 +1,66 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
 /** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+const path = require('path')
+
+console.dir('<rootDir>')
```

Comment:
(Why) Is this needed?

Reply:
its not 4dcb065c73634144073bcb626a8e19d43b66cd07